src/content/deployment/android.md

```diff
@@ -323,11 +323,11 @@ to verify that the values are correct.
 `applicationId`
 : Specify the final, unique [application ID][].
   
-`minSdkVersion`
+`minSdk`
```

Comment:
@bartekpacia I was contemplating if we should tell people that these are replacements for the deprecated properties with similar names. Android Studio at least gives you a warning if you use the old names, so it should be obvious?

Reply:
Yes, I think it's a good idea.

Reply:
I added a `Note` block, like we do for other things on this page. I didn't build a staged version of the docs yet, so I don't know if that looks alright.